### ISSUE TITLE
Commander executeCommand() does not start a terminal on a Windows machine

### ISSUE THREAD

**Issue by wouterbruinings** [Runningriotnl/Vagrant_Launcher#3]

**Comment by Runningriotnl** [comment]
Is this the issue related to the whitespaces in directory names?

**Comment by wouterbruinings** [comment]
No. This is only related to the line of code that executes a command in a terminal.  I am not sure if we can directly reference a line of code from a file in a comment, so I justed copied the code:

MacOS: 
`Runtime.getRuntime().exec(command);`

Windows: 
`Runtime.getRuntime().exec(new String[]{"cmd.exe", command});`

**Comment by Runningriotnl** [comment]
For windows yo do not need a String Array. It something like this:
`Runtime.getRuntime().exec(new String("cmd.exe /c " + command));`
